### PR TITLE
Add KBI on datalad running jupyter notebooks

### PR DESCRIPTION
**Pull request by adswa**

### CODE REVIEW

File: kbi/0003/index.rst [hunk 1]
```diff
@@ -0,0 +1,148 @@
+.. index::
+   single: datalad run; unlock
```
Comment (mih) [inline kbi/0003/index.rst:2]:
```suggestion
.. index:: datalad; run, datalad; unlock
```

Produces
image

**Review by mih — COMMENTED** [review]
This is very useful! Thx!

I have two comments:

I believe the example output file in the code should be called `output.file` are something other than `file` to avoid clashing with the concept of a file, the result type file, etc.

It would make a lot of sense to explicitly state that NBs can be executed like scripts (not just interactively). This is not only a better match for `datalad run`, but also a better match for reproducibility in general.

https://docs.jupyter.org/en/latest/running.html#using-a-command-line-interface

Basically, changing the example to `jupyter run notebook.ipnb`, or leaving it as such, and adding another one.

**Comment by adswa** [comment]
thx, I didn't know about jupyter-run, and must have missed this in the office hour chat. I have added to commits that should address your suggestions, please take look if you have time :)

File: kbi/0003/index.rst [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+.. highlight:: text
+
+KBI0003: Capturing interactive computations in Jupyter Notebooks
+================================================================
+
+:authors: Adina Wagner <a.wagner@fz-juelich.de>
+:discussion: https://github.com/psychoinformatics-de/knowledge-base/pull/1
```
[the remark this answers is not in the document]
Comment (mih) [reply kbi/0003/index.rst:8]: Forgot this, now applied in `main`
